packages/@aws-cdk/aws-cloudfront/README.md

```diff
@@ -33,3 +33,39 @@ const distribution = new CloudFrontWebDistribution(this, 'MyDistribution', {
     ]
  });
 ```
+
+### Viewer certificate
+
+By default, CloudFront Web Distributions will answer HTTPS requests with CloudFront's default certificate, only containing the distribution `domainName` (e.g. d111111abcdef8.cloudfront.net).
+You can customize the viewer certificate property to provide a custom certificate and/or list of domain name aliases to fit your needs.
+
+See [Using Alternate Domain Names and HTTPS](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-https-alternate-domain-names.html) in the CloudFront User Guide.
+
+#### Default certificate
+
+You can customize the default certificate aliases. This is intended to be used in combination with CNAME records in your DNS zone.
+
+Example:
+
+[create a distrubution with an default certificiate example](test/example.default-cert-alias.lit.ts)
+
+#### ACM certificate
+
+You can change the default certificate by one stored Amazon Certificate Manager, or ACM.
+Those certificate can either be generated by AWS, or purchased by another CA imported into ACM.
+
+For more information, see [the aws-certificatemanager module documentation](/FIXME/LINK) or [Importing Certificates into AWS Certificate Manager](https://docs.aws.amazon.com/acm/latest/userguide/import-certificate.html) in the AWS Certificate Manager User Guide.
```

Comment:
Is there a way to cleanly link to another module documentation? Or should I use the relative `aws-certificatemanager-readme.html` link?

Reply:
The only way is a full URL from the reference documentation.

Reply:
Done, thanks